packages/bigtable/src/index.js

```diff
@@ -262,267 +289,262 @@ function Bigtable(options) {
     return new Bigtable(options);
   }
 
-  if (!options.cluster) {
-    throw new Error('A cluster must be provided to interact with Bigtable.');
-  }
-
-  if (!options.zone) {
-    throw new Error('A zone must be provided to interact with Bigtable.');
```

Comment:
If these aren't required anymore, we can remove the lines from the system test that checks if they can run:

```
var clusterName = process.env.GCLOUD_TESTS_BIGTABLE_CLUSTER;
var zoneName = process.env.GCLOUD_TESTS_BIGTABLE_ZONE;

var isTestable = clusterName && zoneName;
```

I believe they are in CONTRIBUTING.md as well, so they need to be removed there as well.